examples/ecs/generic_systems.rs

```diff
@@ -0,0 +1,21 @@
+use bevy::prelude::*;
+use bevy::ecs::component::Component;
+
+// When entities spawn in call an event
```

Comment:
```suggestion
```
This appears to be left over from a copy-paste error perhaps?

Reply:
Oh I had some comments that I was using for notes that one accidentally slipped through.